packages/styles/scss/patterns/sub-patterns/pictogram-item/_pictogram-item.scss

```diff
@@ -11,18 +11,43 @@
 @mixin pictogram-item {
   .#{$prefix}--pictogram-item {
     &__row {
-      @include carbon--make-row();
+      @include carbon--make-row;
     }
-    &__pic-col {
-      margin-top: $carbon--spacing-07;
-      @include carbon--make-col-ready();
-      @include carbon--make-col(2, 16);
+
+    &__wrapper {
+      @include carbon--make-col-ready;
+
+      @include carbon--breakpoint('sm') {
+        @include carbon--make-col(4, 4);
+      }
+
+      @include carbon--breakpoint('md') {
+        @include carbon--make-col(2, 8);
+      }
+
+      @include carbon--breakpoint('lg') {
+        @include carbon--make-col(4, 16);
+      }
     }
-    &__content-col {
-      @include carbon--make-col-ready();
-      @include carbon--make-col(6, 16);
+
+    &__content {
+      @include carbon--make-col-ready;
+
+      @include carbon--breakpoint('sm') {
+        @include carbon--make-col(4, 4);
+      }
+
+      @include carbon--breakpoint('md') {
+        @include carbon--make-col(6, 8);
+      }
+
+      @include carbon--breakpoint('lg') {
+        @include carbon--make-col(12, 16);
+      }
```

Comment:
I'm not sure about all of this grid. I'm thinking it's just two breakpoints within here. `4,4` `@sm`, and `2,8` the rest of the time. Keep in mind this is supposed to sit inside of a container that from this component's point of view is `100%`. From the parent's point of view that's `8,16`.